docs/getting-started.md

```diff
@@ -6,14 +6,14 @@ title: Getting Started
 
 Om aan te sluiten kun je de volgende stappen doorlopen:
 1. [Meld je aan bij het kadaster om toegang te krijgen tot de productieomgeving](#aanmelden-om-aan-te-sluiten)
-2. [Bekijk de functionaliteit en specificaties](#functionaliteit)
-3. [Probeer en test de API in de productie-omgeving](#probeer-en-test-de-api-in-de-productie-omgeving)
-
+2. [Bekijk de functionaliteit en specificaties](#functionaliteit-en-specificaties)
```

Comment:
Wellicht handig om de paragraaf titels in de inhoudsopgave gelijk te houden met de titels van de paragrafen? Dus

2. Bekijk de [functionaliteit en specificaties](#functionaliteit-en-specificaties)
3. [Implementeer API client](#implementeer-api-client]
...

Reply:
Ik ben het deels met je eens.
Het betreft hier nl. niet zozeer een inhoudsopgave maar een stappenplan van waaruit wordt verwezen naar de paragrafen. Vandaar dat de tekst van de stappen mag afwijken van de paragraaftitel. De teksten achter de links moeten we echter wel gelijk maken aan de paragraaftitels.
Eigenlijk is dat ook wat je hierboven voorstelt en wat je denk ik ook bedoelt want je stelt bij 2. immers:

2. Bekijk de [functionaliteit en specificaties](#functionaliteit-en-specificaties)

voor en niet:

2. [functionaliteit en specificaties](#functionaliteit-en-specificaties)

Reply:
Zie mijn nieuwe voorstel.